### ISSUE TITLE
Using asciidoctor js into java Nashorn

### ISSUE THREAD

**Issue by lordofthejars** [asciidoctor/asciidoctor-browser-extension#44]
Hi today I have tried again the integration between asciidoctor js and Java, I think it has been improved so much since last time I tried.

For example rendering Chrome extension README.adoc it has been improved 2 seconds, from 3.5s to 1.5s, JRuby implementation takes 256ms, but I think that with time we will be able to improve a bit more. This time only is rendering time no startup time.

So I decided to open an issue here because I have some questions about opal/asciidoctorjs that I think that may become issues.

So the questions are:
- I have tried to use the method called convert_file to try to convert a file instead of a chunk and next exception is thrown:

javax.script.ScriptException: TypeError: Cannot call undefined in <eval> at line number 12263

Basically it is that input.$readLines doesn't exists.
- I have tried to write an output to disk but without luck, no exception is thrown, I have used to_file attribute.
- I thought about reading the file from Java part and the output write it back, but my next question is if there is a way to get the converted html file as full html (that is with html tag, embedded css, ...) when we pass content as string instead of a file? I have seen that in asciidoctorj some of this information is appended manually.

Well that's all for now we are on the good path :).

Thanks.
